### DISCUSSION TITLE
Maximum number of tags in Loco

### DISCUSSION THREAD

**Discussion by RPeterTech** [bitcraze/discussions#1541] [Q&A] [answered]
How many tags can be used in same Loco positioning group?

**Comment by ataffanel** [comment, marked as answer]
It depends how the loco positioning system is setup: using TWR without TDMA, only one take is supported. Using TDoA modes, there is no limit in the number of tags: when using TDoA, the tags are not transmitting anything in the air and so they do not add any loads to the system.
